dbt/models/reporting/schema.yml

```diff
@@ -6,10 +6,13 @@ sources:
       - name: ratio_stats
         description: '{{ doc("table_ratio_stats") }}'
 
-      - name: res_report_summary
-        description: '{{ doc("table_res_report_summary") }}'
-
 models:
+  - name: reporting.res_report_summary
+    description: '{{ doc("table_res_report_summary") }}'
+    config:
+      tags:
+        - daily
```

Comment:
This tag can be applied to any CTAS-based model we want to rebuild daily. No more dedicated Glue jobs for scheduled rebuilds.